scm-webapp/src/test/java/sonia/scm/io/DefaultContentTypeResolverTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020-present Cloudogu GmbH and Contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package sonia.scm.io;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.nio.charset.StandardCharsets;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class DefaultContentTypeResolverTest {
+
+  private final DefaultContentTypeResolver contentTypeResolver = new DefaultContentTypeResolver();
+
+  @Test
+  void shouldReturnPrimaryPart() {
+    ContentType contentType = contentTypeResolver.resolve("hog.pdf");
+    assertThat(contentType.getPrimary()).isEqualTo("application");
+  }
+
+  @Test
+  void shouldReturnSecondaryPart() {
+    ContentType contentType = contentTypeResolver.resolve("hog.pdf");
+    assertThat(contentType.getSecondary()).isEqualTo("pdf");
+  }
+
+  @Test
+  void shouldReturnRaw() {
+    ContentType contentType = contentTypeResolver.resolve("hog.pdf");
+    assertThat(contentType.getRaw()).isEqualTo("application/pdf");
+  }
+
+  @Nested
+  class IsTextTests {
+
+    @ParameterizedTest(name = "shouldReturnIsTextFor{argumentsWithNames}")
+    @ValueSource(strings = {"App.java", "Dockerfile", "Playbook.yml", "README.md", "LICENSE.txt"})
+    void shouldReturnIsTextFor(String path) {
+      ContentType contentType = contentTypeResolver.resolve(path);
+      assertThat(contentType.isText()).isTrue();
+    }
+
+    @ParameterizedTest(name = "shouldReturnIsNotTextFor{argumentsWithNames}")
```

Comment:
This does not look nice in results

```suggestion
    @ParameterizedTest(name = "shouldReturnIsNotTextFor: {argumentsWithNames}")
```